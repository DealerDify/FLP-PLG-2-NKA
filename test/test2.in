A,B
a,b,c
A
A->aaa