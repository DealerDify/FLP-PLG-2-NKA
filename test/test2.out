1,2,3,4,5
abc
1
5
1,a,3
3,a,4
4,a,5
